src/RNCamera.js

```diff
@@ -1,17 +1,18 @@
 // @flow
 import React from 'react';
+
 import PropTypes from 'prop-types';
 import {
+  ActivityIndicator,
```

Comment:
do we need change order here?

Reply:
Our tooling does this.

Reply:
yes, you are completely right. My attention fell on it because  it's a fork and in general for forks need make less the formatting changing, so that there would be fewer conflicts.